jarbas/frontend/elm/Reimbursement/Update.elm

```diff
@@ -414,34 +407,26 @@ loadReimbursements lang apiKey query =
             jsonQuery =
                 ( "format", "json" ) :: convertQuery query
         in
-            Http.get
-                (url "/api/reimbursement/" jsonQuery)
+            Http.get (url "/api/reimbursement/" jsonQuery)
                 (decoder lang apiKey jsonQuery)
                 |> Http.send LoadReimbursements
 
 
 {-| Convert a list of key/value query pairs to a valid URL:
 
-    >>> toUrl [ ( "year", "2016" ), ( "foo", "bar" ) ]
-    "#/year/2016"
-
-    >>> toUrl [ ( "foo", "bar" ) ]
```

Comment:
So, I've changed the code by removing the `toUrl` function to reduce confusion - or maybe cause more :x

I noticed that the `toUrl` function was only getting called when we had full control of the variables that were being passed to it, therefore we didn't need that isSearchable check. But that wasn't obvious neither guaranteed without an integration test, cause maybe another function could end up using this one, without passing the right params.

I also noted that `toUrl` was always getting called together with `toQuery`, we had to call one to then pass as argument to the other.

So what I did was moving the `toUrl` function to `Search/Update.elm`, since it is only used to build search urls, and now it take the fields as arguments, not query, and return a clean url, so it is no longer possible to pass weird params to it and get a malformed url